File: nested-conditionals/loan_eligibility_check.py
print("=== Loan Eligibility Check ===")
print("-" * 25)

credit_score = int(input("Enter credit score: "))
stable_income = input("Do you have a stable income? (yes/no): ").lower() == "yes"
co_signer_credit = (
    input("Does the co-signer have excellent credit? (yes/no): ").lower() == "yes"
)

if credit_score > 750:
    print("Eligible for the loan.")
elif credit_score >= 650:
    if stable_income:
        print("Eligible for the loan.")
    else:
        print("Not eligible for the loan.")
else:
    if co_signer_credit:
        print("May be eligible for the loan.")
    else:
        print("Not eligible for the loan.")

File: nested-conditionals/loan-eligibility-check.cpp
#include <iostream>
using namespace std;

int main()
{
    cout << "=== Loan Eligibility Check ===" << endl;
    cout << string(25, '-') << endl;

    int credit_score;
    cout << "Enter credit score: ";
    cin >> credit_score;

    string stable_income_input;
    cout << "Do you have a stable income? (yes/no): ";
    cin >> stable_income_input;
    bool stable_income = (stable_income_input == "yes");

    string co_signer_credit_input;
    cout << "Does the co-signer have excellent credit? (yes/no): ";
    cin >> co_signer_credit_input;
    bool co_signer_credit = (co_signer_credit_input == "yes");

    if (credit_score > 750)
    {
        cout << "Eligible for the loan." << endl;
    }
    else if (credit_score >= 650)
    {
        if (stable_income)
        {
            cout << "Eligible for the loan." << endl;
        }
        else
        {
            cout << "Not eligible for the loan." << endl;
        }
    }
    else
    {
        if (co_signer_credit)
        {
            cout << "May be eligible for the loan." << endl;
        }
        else
        {
            cout << "Not eligible for the loan." << endl;
        }
    }

    return 0;
}
